src/main/java/com/google/sps/data/Picture.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.blobstore.BlobKey;
+
+/** A picture. */
+public final class Picture {
+
+  private final long id;
+  private BlobKey blobKey;
+  private String altText;
+
+  public Picture(long id, BlobKey blobKey, String altText) {
+    this.id = id;
+    this.blobKey = blobKey;
+    this.altText = altText;
+  }
+
+  public Picture(EmbeddedEntity embeddedPic) {
+    if (embeddedPic == null) {
+      this.id = -1;
+      return;
```

Comment:
Same here, I'm wondering why this doesn't trigger a compiler error. Shouldn't we return a Picture?